dm/pkg/schema/tracker.go

```diff
@@ -149,7 +149,7 @@ func NewTracker(ctx context.Context, task string, sessionCfg map[string]string,
 		}
 	}
 
-	storePath, err = ioutil.TempDir("./", "schema-tracker")
+	storePath, err = ioutil.TempDir("./", task+"-schema-tracker")
```

Comment:
What about using a sub-directory?

BTW, I think we should add a data-dir configuration for dm-worker. We have relay-dir,dump-dir, and schema-tracker file directories now, but all the default values of them are the dm-worker binary's dir. For TiUP data-dir and deploy-dir are separated which looks a bit weird. 

Maybe we can do this in the next PR.

Reply:
Will do it later, if we can't find the root cause we will change the implementation, so that configuration will be deprecated soon.